sdks/java/core/src/main/java/org/apache/beam/sdk/util/GcsUtil.java

```diff
@@ -386,6 +417,38 @@ public boolean shouldRetry(IOException e) {
      }
   }
 
+  @VisibleForTesting
+  void createBucket(GcsPath path, long projectNumber, BackOff backoff, Sleeper sleeper)
+        throws IOException {
+    Bucket bucket = new Bucket();
+    bucket.setName(path.getBucket());
+    Storage.Buckets.Insert insertBucket =
+      storageClient.buckets().insert(String.valueOf(projectNumber), bucket);
+
+    try {
+      ResilientOperation.retry(
+        ResilientOperation.getGoogleRequestCallable(insertBucket),
+        backoff,
+        new RetryDeterminer<IOException>() {
+          @Override
+            public boolean shouldRetry(IOException e) {
+            if (errorExtractor.itemNotFound(e) || errorExtractor.accessDenied(e)) {
```

Comment:
Just cleanup I hadn't done while hoping we'd resolve the travis issue first.  Sorry about that.